### PR TITLE
docs: add Exherbo Linux package manager to Download instructions

### PR DESCRIPTION
**Pull request by davlgd**
## Description

It adds Exherbo Linux package manager to Download instructions

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: pages/en/download/package-manager/all.md [hunk 1]
```diff
@@ -105,6 +106,14 @@ These resources provide packages compatible with CentOS, Fedora, and RHEL.
 
 Packages compatible with Debian and Ubuntu based Linux distributions are available via [Node.js snaps](#snap).
 
+## Arch Linux
```
Comment (mikeesto) [inline pages/en/download/package-manager/all.md]:
```suggestion
## Exherbo Linux
```

**Review by mikeesto — APPROVED** [review]
LGTM, thanks

**Review by AugustinMauroy — APPROVED** [review]
LGTM !!! Thank for this contribution !

**Review by aymen94 — APPROVED** [review]

**Review by araujogui — APPROVED** [review]
LGTM

**Pull request merged**